Framework/include/QualityControl/CcdbDatabase.h

```diff
@@ -151,6 +150,15 @@ class CcdbDatabase : public DatabaseInterface
    */
   bool isDbInFailure();
 
+  /**
+   * Add the metadata specific to the QC framework.
+   * @param fullMetadata
+   * @param detectorName
+   * @param taskName
+   * @param className
+   */
+  void addQcMetadata(std::map<std::string, std::string>& fullMetadata, std::string detectorName, std::string className);
```

Comment:
Maybe `addFrameworkMetadata`? It was not obvious to me that you meant the QC framework data.